bridget/sap_flow/util.py

```diff
@@ -0,0 +1,10 @@
+def misalignment_correction():
+    pass
+
+
+def wounding_correction():
+    pass
+
+
+def t_stand():
+    pass
```

Comment:
I actually have no idea what these functions should do and what input and output could possibly be